code-for-canada/project-thundercat
# Use Redux to toggle FR/EN images for instructions

# Description

Instead of relying on a localized string (a bit hacky), use Redux to toggle FR/EN images for instructions.

## Type of change
- Code cleanliness or refactor


# Testing


Manual steps to reproduce this functionality:

1.  Go to emib instructions.
2. Toggle language and see images update.

# Checklist

Applicable for all code changes.
